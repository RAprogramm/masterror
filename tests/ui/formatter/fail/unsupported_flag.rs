use masterror::Error;

#[derive(Debug, Error)]
#[error("{value:##x}")]
struct UnsupportedFlag {
    value: u32,
}

fn main() {}
