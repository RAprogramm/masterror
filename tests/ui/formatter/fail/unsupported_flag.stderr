error: placeholder spanning bytes 0..11 uses an unsupported formatter
 --> tests/ui/formatter/fail/unsupported_flag.rs:4:9
  |
4 | #[error("{value:##x}")]
  |         ^^^^^^^^^^^^^

error: missing #[error(...)] attribute
 --> tests/ui/formatter/fail/unsupported_flag.rs:5:8
  |
5 | struct UnsupportedFlag {
  |        ^^^^^^^^^^^^^^^
